### ISSUE TITLE
Error when rebuilding wallet

### ISSUE THREAD

**Issue by brianlenz** [CityOfZion/neo-python#364]
### Current behavior

- What is the problem?

Running `wallet rebuild` results in the following traceback:

```
neo> wallet rebuild
Could not execute command: database is locked
  File ".pyenv/versions/3.6.4/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File ".pyenv/versions/3.6.4/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File ".pyenv/versions/3.6.4/lib/python3.6/threading.py", line 864, in run
    self._target(*self._args, **self._kwargs)
  File "neo-python/venv/lib/python3.6/site-packages/twisted/_threads/_threadworker.py", line 46, in work
    task()
  File "neo-python/venv/lib/python3.6/site-packages/twisted/_threads/_team.py", line 190, in doWork
    task()
  File "neo-python/venv/lib/python3.6/site-packages/twisted/python/threadpool.py", line 250, in inContext
    result = inContext.theWork()
  File "neo-python/venv/lib/python3.6/site-packages/twisted/python/threadpool.py", line 266, in <lambda>
    inContext.theWork = lambda: context.call(ctx, func, *args, **kw)
  File "neo-python/venv/lib/python3.6/site-packages/twisted/python/context.py", line 122, in callWithContext
    return self.currentContext().callWithContext(ctx, func, *args, **kw)
  File "neo-python/venv/lib/python3.6/site-packages/twisted/python/context.py", line 85, in callWithContext
    return func(*args,**kw)
  File "neo/bin/prompt.py", line 987, in run
    traceback.print_stack()
Traceback (most recent call last):
  File "neo-python/venv/lib/python3.6/site-packages/peewee.py", line 3830, in execute_sql
    cursor.execute(sql, params or ())
sqlite3.OperationalError: database is locked

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "neo/bin/prompt.py", line 944, in run
    self.show_wallet(arguments)
  File "neo/bin/prompt.py", line 535, in show_wallet
    self.Wallet.Rebuild()
  File "neo-python/neo/Implementations/Wallets/peewee/UserWallet.py", line 110, in Rebuild
    c.delete_instance()
  File "neo-python/venv/lib/python3.6/site-packages/peewee.py", line 5217, in delete_instance
    return self.delete().where(self._pk_expr()).execute()
  File "neo-python/venv/lib/python3.6/site-packages/peewee.py", line 3614, in execute
    return self.database.rows_affected(self._execute())
  File "neo-python/venv/lib/python3.6/site-packages/peewee.py", line 2939, in _execute
    return self.database.execute_sql(sql, params, self.require_commit)
  File "neo-python/venv/lib/python3.6/site-packages/peewee.py", line 3837, in execute_sql
    self.commit()
  File "neo-python/venv/lib/python3.6/site-packages/peewee.py", line 3656, in __exit__
    reraise(new_type, new_type(*exc_args), traceback)
  File "neo-python/venv/lib/python3.6/site-packages/peewee.py", line 135, in reraise
    raise value.with_traceback(tb)
  File "neo-python/venv/lib/python3.6/site-packages/peewee.py", line 3830, in execute_sql
    cursor.execute(sql, params or ())
peewee.OperationalError: database is locked
```

### Expected behavior

- What should be happening?

Wallet rebuilds without error.

### How to reproduce

- Please explain the steps to reproduce the problem. If you are having an issue with your machine or build tools, the issue belongs on another repository as that is outside of the scope of this project.

I suspect the issue has to do with the fact that my wallet has multiple addresses in it (two smart contract addresses and one "normal" address).

**Comment by brianlenz** [comment]
Dug into this a little bit and realized the issue. It's taking a while to delete the data from the database and while that is occurring, the normal wallet DB loop starts to rebuild, thus resulting in the lock issue. Solution is to stop the wallet DB loop before we start the rebuild and then restart it after.

Working on a PR now...

**Comment by brianlenz** [comment]
This is now fixed in 0.6.5.
